app/javascript/flavours/glitch/styles/accounts.scss

```diff
@@ -573,4 +573,58 @@
   }
 }
 
+.account__header__fields {
```

Comment:
You could reuse the style for `.metadata` instead.

Reply:
Oh, and by reuse, I mean, maybe we could change the name of the existing CSS class to `account__header__fields` and change the glitch-soc-specific code to use that new name.